tests/integration-tests/tests-base/src/main/java/edu/kit/kastel/mcse/ardoco/core/tests/eval/CodeProject.java

```diff
@@ -146,40 +145,18 @@ public String getCommitHash() {
     }
 
     /**
-     * {@return path of the code directory}
-     */
-    public String getCodeLocation() {
-        return getTemporaryCodeLocation().getAbsolutePath();
-    }
-
-    /**
-     * {@return the directory of the code model}
-     */
-    public String getCodeModelDirectory() {
-        try {
-            loadCodeModelFromResourcesIfNeeded();
-            return getTemporaryCodeLocation().getAbsolutePath();
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-    }
-
-    /**
-     * Loads the code from resources or from the code directoy cache
+     * Get Code Location (ACM File or Temporary Directory)
+     * 
+     * @param acmFile If true, the ACM file is loaded from resources
      *
-     * @throws IOException Can occur during file operations
      */
-    public void loadCodeModelFromResourcesIfNeeded() throws IOException {
-        if (ProjectHelper.ANALYZE_CODE_DIRECTLY.get())
-            return;
-
-        File temporaryCodeLocation = getTemporaryCodeLocation();
-        File codeModelFile = new File(temporaryCodeLocation + "/codeModel.acm");
-        try (InputStream is = getClass().getResourceAsStream(this.codeModelLocationInResources)) {
-            try (FileOutputStream fos = new FileOutputStream(codeModelFile)) {
-                is.transferTo(fos);
-            }
+    public File getCodeLocation(boolean acmFile) {
+        if (acmFile) {
+            // If ACM load file from resources
```

Comment:
Minor comment: I think this comment is unnecessary. Is there a specific value it adds?